src/Movie-Card/Movie-Card.scss

```diff
@@ -0,0 +1,77 @@
+@import '../styles.scss';
+
+.movie-card {
```

Comment:
doing a lot of new stuff I'm unfamiliar with here, there are probably some redundancies

Reply:
The screenshot removed my cursor so just imagine a pointer over this:
![Screen Shot 2022-08-28 at 9 51 57 PM](https://user-images.githubusercontent.com/102932448/187119692-2b0eb0d5-3fcc-48e4-bd39-6d5fb771d94b.png)